src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/sentencing/SentencingResourceIntTest.kt

```diff
@@ -764,6 +764,9 @@ class SentencingResourceIntTest : IntegrationTestBase() {
       latestBookingId = prisonerAtMoorland.latestBooking().bookingId
     }
 
+    @AfterEach
+    fun deletePrisoners() = repository.deleteOffenders()
+
```

Comment:
was missing a delete, and because I moved this to new version it detected multiple offenders with the same offenderNo